libs/importer/src/importers/protonpass/types/protonpass-json-type.ts

```diff
@@ -0,0 +1,58 @@
+export type ProtonPassJsonFile = {
+  version: string;
+  userId: string;
+  encrypted: boolean;
+  vaults: Map<string, ProtonPassVault>;
+};
+
+export type ProtonPassVault = {
+  name: string;
+  description: string;
+  display: {
+    color: number;
+    icon: number;
+  };
+  items: ProtonPassItem[];
+};
+
+export type ProtonPassItem = {
+  itemId: string;
+  shareId: string;
+  data: ProtonPassItemData;
+  state: number;
+  aliasEmail: string | null;
+  contentFormatVersion: number;
+  createTime: number;
+  modifyTime: number;
+};
+
+export type ProtonPassItemData = {
+  metadata: ProtonPassItemMetadata;
+  extraFields: ProtonPassItemExtraField[];
+  type: string;
```

Comment:
We should specify which types ProtonPass  currently exports
```suggestion
  type: "login" | "alias" | "creditCard" | "note";
```

Reply:
Added this.